src/sockets/matchmakingQueue.ts

```diff
@@ -65,16 +84,25 @@ export class MatchmakingQueue {
   }
 
   private checkQueue(): void {
-    if (this.queue.length < NUM_PLAYERS_PER_GAME) {
+    if (this.queue.length < MIN_PLAYERS_PER_GAME) {
       return;
     }
 
+    for (
+      let gameSize = MAX_PLAYERS_PER_GAME;
+      gameSize >= MIN_PLAYERS_PER_GAME;
+      gameSize--
+    ) {
+      if (this.checkValidCombinations(gameSize)) {
```

Comment:
checkValidGameSize maybe?

Reply:
Thing is the function is also calling back to sockets so doesn’t feel suitable to call it a check. 

How about TryMatchGameSize